backend/entrypoint.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# shellcheck disable=SC1073
+if [ -f .env ]; then
+  export $(dotenv list -e)
+  fi
+
+echo "Waiting for postgres..."
+
+while ! nc -z 'postgres' $POSTGRES_PORT; do
+  echo "Zzz"
+  sleep 0.5
+done
+
+echo "PostgreSQL started"
+
+
+python manage.py flush --no-input
+python manage.py migrate
+
+if [ "$(python manage.py shell -c "from django.contrib.auth.models import User; print(User.objects.filter(username='admin').exists())")" ]; then
```

Comment:
This does not work with the latest changes. The docker logs say: 
`AttributeError: Manager isn't available; 'auth.User' has been swapped for 'accounts.User'`